README.md

```diff
@@ -25,23 +25,28 @@ Knead Recipes is an app designed to take the hassle and confusion out of what’
 # Features
 
 ## Sign in using Google
-
+![Landing Page](https://i.imgur.com/igWKviA.jpg)
+![Google Sign in](https://i.imgur.com/U6g4wqf.png)
 ---
 ## Set up user profile
-
-
+![Profile Edit](https://i.imgur.com/9cdQ3kf.jpg)
 ---
 ## Enter Ingredients and other Parameters 
-
-
+![Search Page](https://i.imgur.com/Ts7scHx.jpg)
+---
+## Recipie Index
```

Comment:
Typo and I don't think it hurts to add a little snippet on the function of each page. So like "You can enter in ingredients and further filter your results by time,diet,type of meal"

Reply:
it has been done :)